app/helpers/container_summary_helper.rb

```diff
@@ -82,8 +82,7 @@ def textual_guest_applications
 
   def textual_container_image_registry
     object = @record.container_image_registry
-
-    if object.nil?
+    if object.nil? && @record.class == ContainerImage
```

Comment:
We talked about this actually. This function previously assumed only the container image screen called a container image registry relation. This cased an exception for container service with no container image registry attached.